### PR TITLE
Latest Updated Code with Dashboard

### PR DESCRIPTION
**Pull request by harshabonu**
## Issue that this pull request solves

1) Updated Dashboard From gad-5
2) Implemented All Use-Cases.
3) Notification Working successfully 

Closes: # (issue number)

### Brief description of what is fixed or changed

1) Updated Dashboard from gad-5 and Icon for examination module.
2) Completed Submit Grade Use Case 
3) Modified Remaining use cases according to latest suggestions issued
4) Added New features which supports Use Case complete Implementation

## Types of changes

_Put an `x` in the boxes that apply_

-   [ x] Bugfix (non-breaking change which fixes an issue)
-   [x ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [x ] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [ x] I have performed a self-review of my own code
-   [ ] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x ] My changes generate no new warnings
-   [ x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

WhatsApp Image 2024-04-15 at 15 53 44 (1)
WhatsApp Image 2024-04-15 at 15 53 44
WhatsApp Image 2024-04-15 at 15 53 43

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by Priyansh61** [comment]
@harshabonu Kindly completely fill the PR template, helps us understand the PR scope and review faster

**Comment by harshabonu** [comment]
> @harshabonu Kindly completely fill the PR template, helps us understand the PR scope and review faster

@Priyansh61  Kindly, Re-Check the pull request

**Comment by Priyansh61** [comment]
@ChaudharyRaman is reviewing the PR.

File: FusionIIIT/applications/examination/api/views.py [hunk 1]
```diff
@@ -13,194 +12,347 @@
...
+from applications.online_cms.models import(Student_grades)
 from . import serializers
-
+from datetime import date 
 from rest_framework import status
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
 
 
+#Login
```
Comment (ChaudharyRaman) [inline FusionIIIT/applications/examination/api/views.py]: Why so much commented code, Can be removed if not necessary.

**Review by ChaudharyRaman — CHANGES_REQUESTED** [review]
@harshabonu Please remove all the migrations files manually and push.
 Otherwise LGTM @Priyansh61 can be merged.

**Comment by Priyansh61** [comment]
@ChaudharyRaman Is thie functionality working properly? And are there any breaking changes which the module causes to the others?

**Comment by harshabonu** [comment]
@ChaudharyRaman @Priyansh61 
Migration Files are removed and required changes are made.
Kindly, Once again re-check the PR.

**Pull request merged**